src/actor/Attester.spec.ts

```diff
@@ -149,7 +149,7 @@ describe('Attester', () => {
     }
   })
 
-  it('Revoke privacy enhanced attestation', async () => {
+  xit('Revoke privacy enhanced attestation', async () => {
```

Comment:
This test should be fixed before merging. Is there a problem Albi or I should be helping with?

Reply:
Yeah I'd need help with this. Albi told me to open a ticket and disable this instead. Do you have a quick solution? This is also silently failing on develop

Reply:
We will investigate this in a ticket, thanks for telling us!

Reply:
nice! you saw https://github.com/KILTprotocol/ticket/issues/583 right?

Reply:
Yes!